lib/core/src/irods_client_negotiation.cpp

```diff
@@ -43,37 +44,31 @@ namespace irods
     /// @brief given a property map and the target host name decide between a federated key and a local key
     const std::string determine_negotiation_key( const std::string& _host_name )
```

Comment:
I know this isn't part of the main work, but let's remove the `const` from the return type. It forces unnecessary copies.

Reply:
Done